core/services/head_tracker.go

```diff
@@ -399,6 +402,7 @@ func (ht *HeadTracker) subscribe() bool {
 		case <-time.After(ht.sleeper.After()):
 			err := ht.subscribeToHead()
 			if err != nil {
+        promEthConnectionErrors.Inc()
```

Comment:
Weird whitespacing. Maybe your editor has a setting to autoformat on save?

Reply:
fixed - thanks!